jclouds-plugin/src/main/resources/jenkins/plugins/jclouds/compute/JCloudsSlaveTemplate/config.jelly

```diff
@@ -105,28 +105,21 @@
           <f:checkbox />
         </f:entry>
       
-        <f:entry title="Allow Sudo" field="allowSudo">
+        <f:entry title="${%Allow Sudo}" field="allowSudo">
           <f:checkbox />
         </f:entry>
-        <f:entry title="Install Private Key" field="installPrivateKey">
+        <f:entry title="${%Install Private Key}" field="installPrivateKey">
```

Comment:
The documentation refers to *the private key*, needs to be more specific after your change.